test/src/xquery/axes.xml

```diff
@@ -153,7 +153,7 @@
     <test output="text">
         <task>Following axis test: node()</task>
         <code>count(doc("/db/axes-test/axes1.xml")//line/word[. = 'working']/following::node())</code>
-        <expected>9</expected>
+        <expected>17</expected>
```

Comment:
The tests are not doing what they say. If you ask for node() the result will be this. Please remember I am fixing our broken WS handling which has been broken for many many years.